src/services/builder/get-propdate-attestations.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  Attestation,
+  AttestationJsonData,
+  Propdate,
+  PropdateObject,
+} from '@/services/builder/types'
+import { gql, GraphQLClient } from 'graphql-request'
+import { flatMap, pipe } from 'remeda'
+import { attestationsChainsEndpoints } from '.'
+
+interface Data {
+  attestations: Attestation[]
+}
+
+interface Result {
+  propdates: Propdate[]
+}
+
+export const getAttestations = async (): Promise<Result> => {
+  try {
+    const propdatesPromises = attestationsChainsEndpoints.map(
+      async ({ chain, endpoint, schemaId }) => {
+        const query = gql`
+        {
+          attestations(
+            where: {
+              schemaId: {
+                equals: "${schemaId}"
+              }
+              isOffchain: {
+                equals: false
+              }
+            }
+          ) {
+            recipient
+            timeCreated
+            decodedDataJson
+          }
+        }
+      `
+
+        const client = new GraphQLClient(endpoint)
+        const response = await client.request<Data>(query)
+        const propdates = response.attestations
+          .map((attestation) => {
+            const propdateObject = convertPropdateJsonToObject(
+              attestation.decodedDataJson,
+            )
+            return {
+              ...propdateObject,
+              recipient: attestation.recipient,
+              timeCreated: attestation.timeCreated,
+              chain,
+            }
+          })
+          .filter((propdate) => !propdate.replyTo) // Filter out propdates with non-empty replyTo
+
+        return propdates
+      },
+    )
+    const results = await Promise.all(propdatesPromises)
+
+    const propdates = pipe(
+      results,
+      flatMap((propdates) => propdates),
+    )
+
+    console.log(propdates)
```

Comment:
There are some occurences of `console.log`, which can be either removed or replaced with `logger.debug`